{
    "/home/zhang/Packages/httpx/httpx0.11.0/utils.py": [
        {
            "httpx.utils.Logger.trace": 221
        },
        {
            "httpx.utils.trace": 251
        },
        {
            "httpx.utils.as_network_error": 362
        }
    ],
    "/home/zhang/Packages/httpx/httpx0.11.0/exceptions.py": [
        {
            "httpx.exceptions.HTTPError.__init__": 12
        }
    ],
    "/home/zhang/Packages/httpx/httpx0.11.0/models.py": [
        {
            "httpx.models.URL.copy_with": 196
        },
        {
            "httpx.models.QueryParams.__init__": 291
        },
        {
            "httpx.models.Response.json": 845
        }
    ]
}